backend/src/services/master/master.service.ts

```diff
@@ -70,14 +69,14 @@ class Master {
       },
       token: this.#tokenService.create({
         userId: id,
+        userRole: UserRole.master,
       }),
     };
   }
 
   public async getCurrentUser(
-    token: string,
+    userId: string,
   ): Promise<EAMMasterSignUpResponseDto> {
```

Comment:
This change will break backend\src\api\masters\masters.api.ts:53 - `.send(await masterService.getCurrentUser(token))`. So, you can either roll this changes back or remove /user route from master api. 
Also, `getCurrentUser` method which takes `userId` as a param bothers me. I'd rename it to `getUserById(userId: string)`

Reply:
I think so.